ray-operator/controllers/ray/common/pod.go

```diff
@@ -711,6 +711,14 @@ func addEmptyDir(container *v1.Container, pod *v1.Pod, volumeName string, volume
 	if checkIfVolumeMounted(container, pod, volumeMountPath) {
 		return
 	}
+
+	// 0) Skip adding the volume if it is already in the list of volumes
+	for _, volume := range container.VolumeMounts {
+		if volume.Name == volumeName && volume.MountPath == volumeMountPath {
+			return
+		}
+	}
```

Comment:
Thanks for fixing this!

Would you mind factoring this out into a function `checkIfVolumeMountExists` similar to the `checkIfVolumeExists` function just below it?

Reply:
sure. let me first test it with @iycheng and see if this resolves the issue.

Reply:
Agree. This should be easily verified in local env.